packages/utilities-deprecated/projects/utilities/CHANGELOG.md

```diff
@@ -9,6 +9,524 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### [Unreleased]
 
+This package is **deprecated** and will no longer be maintained. All `Porsche Design System` utilities are now provided
+at`@porsche-design-system/components-{js|angular|react|vue}/styles` in from of styles and Design Tokens. To make the
+migration easier, we offer an overview of the old deprecated values in reference to the new design tokens. Further
+documentation about the new Design Tokens can be found here
+[Introduction](https://designsystem.porsche.com/latest/styles/introduction).
+
+#### Changed
+
+##### JS
+
+**Functions**
+
+- `focus()` is now `getFocusStyles()`. The parameters have changed as the focus is now handled with `border` and rounded
+  edges. You can choose `small` or `medium` for the `borderRadius`. `getFocusStyles()` also excepts an `theme` parameter
+  instead of a custom `color`, which determines the color of the focus.
+
+```diff
+- import { focus } from '@porsche-design-system/utilities'
++ import { getFocusStyles } from '@porsche-design-system/components-{js|angular|react|vue}/styles'
+```
+
+- `breakpoint` `xxs` is now `base`. The values have not changed.
+- Instead of `mediaQuery()` we provide now `getMediaQueryMin`, `getMediaQueryMax` and `getMediaQueryMinMax`.
+  Furthermore, the functions accept only the predefined PDS breakpoints from `base` to `xxl` and no custom breakpoints
+  anymore.
+
+```diff
+- import { breakpoint, mediaQuery } from '@porsche-design-system/utilities'
++ import { breakpoint, getMediaQueryMin, getMediaQueryMax, getMediaQueryMinMax } from '@porsche-design-system/components-{js|angular|react|vue}/styles'
+```
+
+- `titleLarge` got renamed to `displayLargeStyle`. The size got adjusted. In addition, we also provide
+  `displayMediumStyle`. The new `display` styles differ also, that they are recursive and their size is changing linear
+  fluid depending on the viewport width instead of fixed predefined sizes on specific breakpoints.
```

Comment:
I think linear fluid isn't true. I would just call it fluid because it doesn't scale or shrink infinite. The opposite would be static or dynamic with breakpoints. Fixed is kind of confusing.